content/en-us/cloud-services/memory-stores/best-practices.md

```diff
@@ -34,13 +34,13 @@ Use a helper function to get the correct sorted map from an item key. This way,
 
 ```lua title='Sharding a Sorted Map'
 -- Initialize the MemoryStore Service
-local memoryStore = game:GetService("MemoryStoreService")
+local MemoryStore = game:GetService("MemoryStoreService")
```

Comment:
```suggestion
local MemoryStoreService = game:GetService("MemoryStoreService")
```

Reply:
Just matching the variable name to the service name, which we commonly like to do in docs.